api/filters.go

```diff
@@ -381,6 +427,29 @@ func (api *API) getCantabularDimension(ctx context.Context, filterID, dimensionN
 	return &foundDimensions.Dataset.Variables.Search.Edges[0].Node, nil
 }
 
+// getCantabularDimension checks that dimension exists in Cantabular by searching for it.
+// If the dimension doesn't exist, or couldn't be retrieved, an error is returned.
+func (api *API) getCantabularDimensionNoFilterID(ctx context.Context, popType, dimensionName string) (*gql.Node, error) {
```

Comment:
Can we please remove getCantabularDimensionNoFilterID and update  getCantabularDimension where population_type is passed as parameter rather than filter_id

Reply:
done